frameworks/kafka/docs/upgrade.md

```diff
@@ -1,25 +1,25 @@
 ---
-post_title: Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+post_title: Upgrade Kafka from 0.10.2.x to 0.11.0.0
 menu_order: 10
 feature_maturity: preview
 enterprise: 'no'
 ---
 
-# Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+This document explains how to upgrade the DC/OS Kafka service from version 0.10.2.x to 0.11.0.0.
```

Comment:
We should be explicit about the version you can upgrade from.  Please revise the rest of the document with this as well.